_aicspylibczi/DimIndex.h

```diff
@@ -43,13 +44,16 @@ enum class DimIndex : std::uint8_t
  */
 char
 dimIndexToChar(DimIndex index_);
+
 DimIndex
 charToDimIndex(char c_);
-libCZI::DimensionIndex
-dimIndexToDimensionIndex(DimIndex index_);
+
 DimIndex
 dimensionIndexToDimIndex(libCZI::DimensionIndex index_);
 
+libCZI::DimensionIndex
+DimIndexToDimensionIndex(DimIndex dim_index_);
```

Comment:
why did this function name get capitalized?

Reply:
I changed it to be consistent. Thanks for pointing it out.